libraries/AP_Parachute/AP_Parachute.h

```diff
@@ -72,11 +88,15 @@ class AP_Parachute {
     AP_Int16    _servo_off_pwm; // PWM value to move servo to when shutter is deactivated
     AP_Int16    _alt_min;       // min altitude the vehicle should have before parachute is released
     AP_Int16    _delay_ms;      // delay before chute release for motors to stop
+    AP_Int8     _emergency_enabled;  // 1 if emergency parachute release is enabled
+    AP_Float    _max_sink;      // maximum sink rate to trigger emergency parachute
 
     // internal variables
     AP_Relay   &_relay;         // pointer to relay object from the base class Relay.
     uint32_t    _release_time;  // system time that parachute is ordered to be released (actual release will happen 0.5 seconds later)
     bool        _release_initiated:1;    // true if the parachute release initiated (may still be waiting for engine to be suppressed etc.)
     bool        _release_in_progress:1;  // true if the parachute release is in progress
     bool        _released:1;             // true if the parachute has been released
+    bool        _is_flying;              // true if the vehicle is flying
+    AP_Float    _sink_rate;              // vehicle sink rate in m/s
```

Comment:
Why is this an `AP_Float` instead of just `float`? This isn't used in the param table anywhere.